
int16 gps_week
uint32 gps_time
uint8 imu_alignment_status
uint8 gnss_status
